src/writeData/constants/contentFileUploads.ts

```diff
@@ -22,21 +22,21 @@ export const WRITE_DATA_FILE_UPLOADS: FileUpload[] = [
     image: CSVLogo,
     markdown: AnnotatedCSVMarkdown,
   },
-  {
-    id: 'csv',
-    name: 'CSV Data',
-    image: CSVLogo,
-    markdown: CSVMarkdown,
-  },
   {
     id: 'lp',
     name: 'Line Protocol',
     image: LPLogo,
     markdown: LPMarkdown,
   },
+  {
+    id: 'csv',
+    name: 'CSV Data (CLI)',
+    image: CSVLogo,
+    markdown: CSVMarkdown,
+  },
 ]
 
 export const search = (term: string): FileUpload[] =>
   WRITE_DATA_FILE_UPLOADS.filter(item =>
     item.name.toLowerCase().includes(term.toLowerCase())
-  ).sort((a, b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()))
```

Comment:
We only have three choices, we can force an order for them, which we want to do with the CSV data tile.